planner/core/src/main/java/eu/seaclouds/platform/planner/core/Planner.java

```diff
@@ -296,32 +309,31 @@ public SLAInfo() {}
 
     }
 
-    class OfferJson {
+    public static class Offering {
```

Comment:
This is exactly the problem I'm trying to avoid here. We should have a single domain object that represents the same concept.